src/SFML/Graphics/Image.cpp

```diff
@@ -328,10 +328,10 @@ void Image::flipVertically()
 {
     if (!m_pixels.empty())
     {
-        std::vector<Uint8>::iterator::difference_type rowSize = static_cast<std::vector<Uint8>::iterator::difference_type>(m_size.x * 4);
+        auto rowSize = static_cast<std::vector<Uint8>::iterator::difference_type>(m_size.x * 4);
```

Comment:
I know it's technically correct, but this will always be `std::ptrdiff_t`, no?
Should we use that to help readability?

Reply:
Hmm, I wouldn't risk it to be honest. We support many platforms and I would have to ensure that all of them do indeed use `std::ptrdiff_t`. I'd rather be "technically correct" in this case.